donate/payments/tasks.py

```diff
@@ -142,13 +147,14 @@ def process_subscription_charged_successfully(self, notification):
             'transaction_id': last_tx.id,
             'subscription_id': notification.subscription.id,
             'project': custom_fields.get('project', ''),
+            'card_type': 'Unknown' if is_paypal else last_tx.credit_card_details.card_type,
```

Comment:
At first glance, this seems like it might break when we start accepting other payment types. (I'm guessing they won't all have `credit_card_details`, but I could be wrong.) Can we make it a bit more defensive?

Reply:
I would recommend we do this in followup, then, because the line right below this will also be affected by any "not-paypal, not-creditcard" transaction, and I based this line off of that one.

Reply:
Fair enough